frontend/src/Views/Footer.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @desc A footer with links to our terms of agreement and BC NMP Help resource
+ *@author @GDamaso
+ */
+import StyledFooter from './Footer.style';
+
+const Footer = () => (
+  <StyledFooter>
+    <p>
+      By continuing, you agree to our&nbsp;
+      <a href="https://nmp.apps.nrs.gov.bc.ca/Information/Disclaimer">Terms of Service</a>
+    </p>
+    <p>
+      Need&nbsp;
+      <a href="https://www2.gov.bc.ca/gov/content/industry/agriculture-seafood/agricultural-land-and-environment/soil-nutrients/nutrient-management/nutrient-management-calculator/support-for-the-bc-nutrient-management-calculator">
```

Comment:
Suggestion: Might be valuable to move your URLS into variables held in the component
1. Simplifies the `tsx`
2. Adds intent to what the urls are for (These urls are pretty self-explanatory though)